developing-with-nats/security/creds.md

```diff
@@ -80,11 +79,11 @@ await nc.close()
 {% tab title="TypeScript" %}
 ```typescript
 // credentials file contains the JWT and the secret signing key
-let credsFile = path.join(confDir, 'credsfile.creds');
+const data = fs.readFileSync(creds_file_path);
```

Comment:
```suggestion
const data = fs.readFileSync(credsFile);
```

Reply:
I would expand on the comment to say that reading of the credentials file will be different depending on the JavaScript runtime, and that the code shown there is for node.js